tests/settings/ConfigurationGatheringTest.py

```diff
@@ -119,8 +122,6 @@ def test_system_coafile_parsing(self):
         Constants.system_coafile = tmp
```

Comment:
what about this one?

Reply:
This is `system_coafile`which is  different from `user_coafile`

Reply:
@jayvdb ping

Reply:
@siddhant-curious 
The ci is red because you have not followed our commit guidelines
(your maximum line length in body and issue references are inappropriate)

Please fix them before requesting further reviews

Reply:
@virresh okay, I will fix it now.

Reply:
done

Reply:
@siddhant-curious Referencing #5448 `If the assertEqual fails, Constants.user_coafile will not be restored, which could result in lots of other tests to fail in unpredictable ways.`. The same applies for all areas where `Constants.system_coafile` is used .

Reply:
Yes it does. This is one of the things I was asking on the channel. I obviously have to fix it; on it.

Reply:
Done.